dll.py

```diff
@@ -49,4 +49,3 @@ def remove(self, node):
             self._first_node = node.next
         if self._last_node is node:
             self._last_node = node.prev
-
```

Comment:
Interessant, ik had ergens anders geleerd dat je files juist met een lege lijn moest eindigen.

Reply:
Dat is de POSIX standaard voor bestanden: elke regel eindigt met een newline character. Dus bijv. een leeg bestand bevat een newline character, want het bestand bevat een lege regel. De laatste regel een niet leeg bestand eindigt dus ook met een newline character.

Git vindt het leuk dat je dat doet, maar het is niet vereist. Echter, je ziet het overal (bijv. GitHub diff of `git diff`) dat er een newline character 'ontbreekt'.